shared/src/persistence/dynamo/cases/updateCase.test.js

```diff
@@ -460,6 +502,208 @@ describe('updateCase', () => {
     });
   });
 
+  describe('archivedCorrespondences', () => {
+    it('adds an archived correspondence to a case when archivedCorrespondences is an empty list', async () => {
```

Comment:
Is `archivedCorrespondences` supposed to be an empty list in this test? It doesn't appear that way in the updateCase call below.

Reply:
I can see how this could cause confusion! The idea was for this test that the mocked out case record (the record in dynamo) does not have any entries in archivedCorrespondences. The updateCase method is then called with caseToUpdate having an entry in archivedCorrespondences as a test of the diff being computed in the method under test. Would it make it more clear to add an expectation before the method is called verifying the case record archivedCorrespondences is empty? (same for correspondence and documents as commented on below)

Reply:
Yeah, that would make it more clear! Thank you for the explanation 👍